pkgs/applications/networking/instant-messengers/trillian-im/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, dpkg
+, atkmm
+, cairo
+, cairomm
+, gtk3
+, gtkmm3
+, libnotify
+, libsecret
+, pangomm
+, xorg
+, libpulseaudio
+, librsvg
+, libzip
+, openssl
+, webkitgtk
+, libappindicator-gtk3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "trillian-im";
+  version = "6.3.0.1";
+
+  src =
+    if stdenv.hostPlatform.system == "x86_64-linux" then
+      fetchurl {
+        url = "https://www.trillian.im/get/linux/6.3/trillian_${version}_amd64.deb";
+        sha256 = "42e3466ee236ac2644907059f0961eba3a6ed6b6156afb2c57f54ebe6065ac6f";
+      }
+    else
+      throw "Trillian for Linux is not supported on ${stdenv.hostPlatform.system}";
```

Comment:
I don't think we need this. platform is set accordingly and this is only required when using attrs because otherwise we get eval errors on other platforms.

Reply:
Thanks - I removed the check.